content/tutorials/single-page-tutorials/mint-nft-with-javascript-shimmer.md

```diff
@@ -0,0 +1,239 @@
+---
+title: How to mint a native NFT with Javascript on Shimmer
+description: Learn how to mint an NFT on the Shimmer Network with Javascript
+tags: [Shimmer, NFT, beginner, Javascript, iota.js, client]
+---
+
+# How to Mint a Native NFT With Javascript on Shimmer
+
+A non-fungible token (NFT) is a token that is verifiably unique and can be used to prove and transfer ownership of a digital asset. You can create NFTs in the Shimmer ledger as a native asset or use smart contracts as a smart asset. This tutorial uses Javascript to mint a native NFT in the Shimmer ledger.
```

Comment:
Good catch about "smart asset". This was a term used a few months back within IF, but it has no mention at all on the Wiki, so I guess we should ditch it. Also I might have misused the term "native asset" here in that case, as I was using it in comparison to "smart asset". What would be the correct terms here? L1 and L2 native assets? The distinction being in the layer?

Reply:
I was trying to avoid using "L1" and "L2" in this case, because those terms might need some explanation to new comers. I thought it would be best explicitly mentioning NFTs can be created either in the Shimmer ledger or in a smart contract ledger.